### ISSUE TITLE
Test TracIn's effectiveness in text classification

### ISSUE THREAD

**Issue by yananchen1989** [frederick0329/TracIn#8]
Hello,

I adopt the code from https://github.com/frederick0329/TracIn/blob/master/imagenet/resnet50_imagenet_proponents_opponents.ipynb
to text classification, where the classifier is Albert based.

The primary goal of  my task is to evaluate the positive or negative impacts on the validation set. The core metrics can be accuracy or cross entropy loss for my task. Quite straightforward. Where the training samples could be 100-200 and validation set contains no more than 100 samples. This is a low-data regime.

Validation set is of no error. It Is clean.

Labels include politics business tech entertainment etc. Just a public news topic classification task: AG NEWS.
 
As for the classifier, similar to your resnet in the image example, I am using ALBERT. The checkpoints are albert classifiers. As you know, bert Is of large trainable parameters. Following your setting, I just take the last two layers to compute the gradients.
 That means the bert part will be frozen when getting the influence scores.  

here is my implementation
https://github.com/yananchen1989/topic_classification_augmentation/blob/main/albert_proponents_opponents.py

I use AUC in the last, to test the effectiveness: high auc indicate that samples of no labelling noise get higher influence score, while samples wrongly labelled, get lower, negative score. 
However, the auc is 0.55. Quite woeful.

I am not sure if there is a bug in my implementation, or I have not using TracIn in a appropriate manner.

**Comment by yananchen1989** [comment]
I intentionally fabricate some samples whose labels are incorrect, `df_noise`. It can be regarded as honeypot to test TracIn's ability to score them. It is anticipated that TracIn should give them significantly negative influence scores, and on the other hand, TracIn should give higher positive scores to samples from `df_train` since it is absolutely clean.

I assume that `df_noise` should acts as the opponents, with regard to the `df_valid` on the whole, which is also a clean set.

**Comment by yananchen1989** [comment]
For the checkpoints picking, currently, I pick those checkpoints which reduce the loss on validation set obviously. 
I set the total epochs to 50. and checkpoints from early epochs are more likely to be saved, because reduction in loss in previous epochs are more likely and tangible.

Generally, I would collect 10-15 checkpoints for following loss calculation.

**Comment by Jiahaou** [comment]
请问源代码中的ckpt文件在哪里下载
